bundles.rst

```diff
@@ -98,6 +98,9 @@ to be adjusted if needed:
 ``src/``
     Contains all PHP classes related to the bundle logic (e.g. ``Controller/RandomController.php``).
 
+``assets/``
```

Comment:
What about putting it after `public`?

Reply:
I agree. I did that change while merging. Thanks.